tests/simple/testdata/integer_math.textproto

```diff
@@ -1,2 +1,171 @@
 name: "integer_math"
 description: "Tests for int and uint math."
+section {
+  name: "int64_math"
+  description: "Simple tests for int64."
+  test {
+    name: "simple_add_positive_positive"
+    expr: "42 + 2"
+    value: { int64_value: 44 }
+  }
+  test {
+    name: "simple_add_positive_negative"
+    expr: "42 + (-2)"
```

Comment:
Change the arguments around a little.  It's still okay to use small numbers.

Reply:
done